src/style.scss

```diff
@@ -32,7 +32,7 @@
   border-color: transparent;
 
   background-color: var(--auro-color-background-lightest);
-  color: var(--auro-color-text-secondary-on-light);
+  color: var(--auro-color-text-primary-on-light);
```

Comment:
We do not want to make this change because this will change the initial placeholder value color, which is supposed to be `--auro-color-text-secondary-on-light`. Refer to this previous PR:
- https://github.com/AlaskaAirlines/auro-dropdown/pull/177

Reply:
Disregard the previous statement, this is indeed the correct change.